src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -102,106 +123,168 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
+                              .action=${stateObj.state === "off"
+                                ? "turn_on"
+                                : "turn_off"}
+                              @click=${this._handleClick}
+                            ></paper-icon-button>
+                          `}
+                    </div>
+                    ${OFF_STATES.includes(stateObj.state)
+                      ? ""
+                      : html`
+                          <div class="playback-controls">
+                            ${!supportsFeature(stateObj, SUPPORT_PREVIOUS_TRACK)
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    icon="hass:skip-previous"
+                                    .action=${"media_previous_track"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                            ${(stateObj.state !== "playing" &&
+                              !supportsFeature(stateObj, SUPPORTS_PLAY)) ||
+                            stateObj.state === UNAVAILABLE ||
+                            (stateObj.state === "playing" &&
+                              !supportsFeature(stateObj, SUPPORT_PAUSE) &&
+                              !supportsFeature(stateObj, SUPPORT_STOP))
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    class="playPauseButton"
+                                    .icon=${stateObj.state !== "playing"
+                                      ? "hass:play"
+                                      : supportsFeature(stateObj, SUPPORT_PAUSE)
+                                      ? "hass:pause"
+                                      : "hass:stop"}
+                                    .action=${"media_play_pause"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                            ${!supportsFeature(stateObj, SUPPORT_NEXT_TRACK)
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    icon="hass:skip-next"
+                                    .action=${"media_next_track"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                          </div>
+                        `}
+                  </div>
                 `}
           </div>
-          <paper-icon-button
-            icon="hass:dots-vertical"
-            @click=${this._handleMoreInfo}
-          ></paper-icon-button>
+          ${OFF_STATES.includes(stateObj.state) ||
```

Comment:
You might want to put this "showProgressBar" in a getter, as you'll also need it in `updated` to schedule an update every second to update to the latest progress if playing.